### ISSUE TITLE
as a funder, i want feedback / commenting ability on the accept submission page, so i can give feedback

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#1410]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
feedback / commenting ability
as a funder, i want feedback / commenting ability on the accept submission page, so i can give feedback

The modal suggests doing this, but its not in the UI anywhere!
http://bits.owocki.com/1F361J1J1Y2l/Screen%20Shot%202018-06-08%20at%209.45.30%20AM.png

### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
so we can collect feedback on how it went.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: 
feature

### Success Criteria

When a user submits a bounty fulfillment OR when a funder performs a basic payout, we would like to add the following elements to the form.

- num stars (0 to 5)
- public comments
- private comments

these will then be sent to the blockchain as part of the fulfillment (if fulfillment tx) or payout (if payout tx).. 

when the system sees a new review of a bounty hunter/funder in the system, it will do the following

- store in the backend as a `UserReview`
- aggregate the reviews and show an average star rating for that user on their profile
- attach it to the 'work done' email which is sent to both parties

**Comment by PixelantDesign** [comment]
@owocki do you already have a PR out for this?

**Comment by owocki** [comment]
yes but it is WIP

**Comment by PixelantDesign** [comment]
Need @mbeacom  to scope

**Comment by owocki** [comment]
Is this blocked by organizations, or can it be done in parallel?  

Since we're having a hard time finding bandwidth for it, should we just bounty it?

**Comment by PixelantDesign** [comment]
I’m okay with bountying.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.3 ETH (61.45 USD @ $204.84/ETH)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1410/1216).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/1410/1216).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $21,634.56 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 4 weeks from now.
Please review their action plans below:


**1) [mkjekk](https://gitcoin.co/profile/mkjekk) has started work.**

To Get lms startet and make some seriøs monny

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/1410/1216).

**Comment by owocki** [comment]
@mkjekk hows it going?  any update!?

**Comment by mbeacom** [comment]
🤔 Should I drop my branch for this? @owocki

**Comment by owocki** [comment]
@mbeacom i dont know.. would be good to get orgs up on staging so we can test/see whats in there

**Comment by mbeacom** [comment]
@owocki Roger that. I'll stop pursuing the reputation PR and the other bountied PRs that we're trying to wrap up needing core intervention and get orgs up there for ya.

**Comment by gitcoinbot** [comment]
@mkjekk Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=100)

**Comment by gitcoinbot** [comment]
@mkjekk Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/1410/1216?snooze=100)

**Comment by owocki** [comment]
@mbeacom since the contributor is MIA, and since you have a PR open for similar functionality... im caneling / closing this bounty

**Comment by owocki** [comment]
@mbeacom let me know if i should move the issue details to another new issue?  i havent tested orgs yet so i dont know if this is really in scope / tackled by it

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 0.3 ETH  (70.86 USD @ $236.2/ETH) attached to this issue has been **cancelled** by the bounty submitter__

 
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $23,213.13 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
